master/master.cfg

```diff
@@ -722,31 +722,173 @@ def get_cmake_build_command(builder_type, build_dir, targets=None):
     return cmd
 
 
+# options in LLVM that we always want OFF because they aren't needed
+# for Halide buildbot usage.
```

Comment:
This comment is wrong and misleading. This doesn't just affect the LLVM used on the buildbots. This is also the LLVM that gets bundled into our libHalide binary releases. Not building the tools has no effect on that, but these decisions are decisions we're making for downstream users of libHalide, not just on our buildbots.

Reply:
I'm going to fix it in https://github.com/halide/build_bot/pull/276